lib/validator/validator.go

```diff
@@ -33,7 +34,7 @@ var validate *validator.Validate
 var (
 	nameRegex          = regexp.MustCompile("^[a-zA-Z0-9_.-]+$")
 	labelRegex         = regexp.MustCompile("^[a-zA-Z_./-][a-zA-Z0-9_./-]*$")
```

Comment:
This is less permissive than felix's regex, which allows numerics in the first char.  No harm in being more aggressive here.